pytorch_lightning/loggers/neptune.py

```diff
@@ -337,9 +343,12 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
             neptune_logger.log_hyperparams(PARAMS)
         """
         params = self._convert_params(params)
-        params = self._flatten_dict(params)
-        for key, val in params.items():
-            self.run[f'param__{key}'] = val
+
+        parameters_key = self.PARAMETERS_KEY
+        if self._base_namespace:
+            parameters_key = f'{self._base_namespace}/{parameters_key}'
+
+        self.run[parameters_key] = params
```

Comment:
Just do `self.run[self._base_namespace][parameters_key] = params` assuming that `self._base_namespace` is an empty string by default

Reply:
ok

Reply:
One the other hand it will make tests significantly more complicated and ugly, are you sure that we don't want to keep this internal if?